WordPress/src/main/java/org/wordpress/android/widgets/MySiteTitleAndSubtitleLabelView.kt

```diff
@@ -36,7 +36,7 @@ class MySiteTitleAndSubtitleLabelView @JvmOverloads constructor(
                 constraintSet.clone(this@MySiteTitleAndSubtitleLabelView)
                 constraintSet.connect(title.id, ConstraintSet.BOTTOM, guideline.id, ConstraintSet.TOP, 0)
                 constraintSet.applyTo(this@MySiteTitleAndSubtitleLabelView)
-            } else if (title.lineCount == 2 && (title.layoutParams as LayoutParams).bottomToTop == guideline.id) {
+            } else if (title.lineCount  >    1 && (title.layoutParams as LayoutParams).bottomToTop == guideline.id) {
```

Comment:
Minor [fomatting issue:](https://buildkite.com/wordpress-mobile/wordpress-android/builds/1299#8b0b0c36-6d7c-4e28-be3a-0a9bb430a942/156-203) 

```suggestion
            } else if (title.lineCount > 1 && (title.layoutParams as LayoutParams).bottomToTop == guideline.id) {
```

Reply:
@ashiagr  Thanks for the help. Resolved the formatting issue.